environments/prod/host_vars/backup2.cnx.org

```diff
@@ -0,0 +1,4 @@
+---
+slim_dumps: yes
+# Point at the replicant:
+archive_db_host: prod10.cnx.org
```

Comment:
slim dump script and replicant are not compatible.

Reply:
The backup script using the `archive_db_host` value here. The database dump to the backup server is being created from the database replicant.

Reply:
Ah, I see now - it's a two step: backup uses `archive_db_host` to make a local copy via `pg_dump`, then slim dump is created locally on backup server.